Source/RapyutaSimulationPlugins/Public/Core/RRUObjectUtils.h

```diff
@@ -40,13 +42,15 @@ class RAPYUTASIMULATIONPLUGINS_API URRUObjectUtils : public UBlueprintFunctionLi
     /**
      * @brief Use CreateDefaultSubobject or NewObject based on where this method is called.
      * Uses #URRThreadUtils::IsInsideConstructor.
-     * @param InOuter 
-     * @param InObjectClass 
-     * @param InObjectUniqueName 
+     * @param InOuter
+     * @param InObjectClass
+     * @param InObjectUniqueName
      * @return static UObject*
      *
-     * @sa [CreateDefaultSubobject](https://docs.unrealengine.com/4.27/en-US/API/Runtime/CoreUObject/UObject/UObject/CreateDefaultSubobject/2/)
-     * @sa [NewObject](https://docs.unrealengine.com/4.27/en-US/ProgrammingAndScripting/ProgrammingWithCPP/UnrealArchitecture/Objects/Creation/)
+     * @sa
```

Comment:
shouldn't @sa and later in the same line?

Reply:
It seems clang-format auto break the long line

Reply:
Can you check it works or not and if it not works, can you avoid change line?

Reply:
ah I see.